docs/services/data/configure-data-capture.md

```diff
@@ -238,7 +245,7 @@ The following example captures data from two analog readers that provide a volta
               {
                 "method": "Gpio",
                 "capture_frequency_hz": 1,
-                "name": "rdk:component:board/board",
+                "name": "rdk:component:board/my-esp32",
                 "additional_params": {
                   "27": ""
```

Comment:
This should be grabbing the GPIO pins by name (again, this will be more clear with the refactor we're making now & I'll update the docs after), but then this should be "15"

Under the hood, this is calling [GPIOPinByName](https://github.com/viamrobotics/rdk/blob/d292ecc835cab4dc88c2e51e6529d40766111319/components/board/board.go#L62)

Reply:
I was thinking 15 is the number of pins as opposed to a pin number, but I'm not sure where to find the info on what the esp32 config stuff even does/means

Reply:
I would ask Nicolas about that. I think it's also completely reasonable to have a non-ESP32 board example here.

Reply:
changed it to 27 above